src/lib/config.js

```diff
@@ -1,5 +1,3 @@
-require("dotenv").config();
```

Comment:
This is needed to local dev please can we add it back in

Reply:
Readded